homeassistant/components/fritz/switch.py

```diff
@@ -15,13 +15,14 @@
 )
 import xmltodict
 
+from homeassistant.components.network import async_get_source_ip
```

Comment:
We should have `network` as a dep in `manifest.json`

Reply:
Is already dependant: https://github.com/home-assistant/core/blob/450fdc91e47fe9cf76224db8141098e265ad0b79/homeassistant/components/fritz/manifest.json#L9